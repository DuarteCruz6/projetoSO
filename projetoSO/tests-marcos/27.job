BACKUP
WRITE [(r5,14)(c7,19)]
WRITE [(z1,13)(b5,50)]
READ [r5]
WRITE [(v6,62)(q0,100)(f4,90)]
WRITE [(u7,32)(f5,65)(w0,50)]
WRITE [(b9,20)(x7,99)]
WRITE [(r7,83)(l8,67)(l5,26)(c9,73)]
WRITE [(a4,34)(m2,62)(m3,21)(w9,59)]
WRITE [(o4,77)(k1,11)(k1,47)]
BACKUP
READ [k1]
DELETE [w9,k1,l5,x7]
WRITE [(d8,93)(m4,99)(g6,63)(a2,89)]
WAIT 1000
WRITE [(d4,21)(l2,72)(m8,24)(q2,29)]
WRITE [(p1,14)(d7,15)(s4,82)(i8,78)]
READ [z1]
SHOW
WAIT 2000
WRITE [(v2,57)(y3,15)(z2,30)]
DELETE [o4,b5,m4]
DELETE [b9,r7,z1,u7,v6]
WAIT 1000
DELETE [q0,a2,w0]
READ [l2]
WRITE [(r7,38)(j3,26)(v5,34)]
BACKUP
WRITE [(q8,91)]
WRITE [(d5,56)(u7,61)(e3,77)(d3,7)(n7,28)]
WRITE [(o2,26)(n8,55)(o9,68)]
DELETE [j3,g6,m8]
WRITE [(k5,71)(s9,34)(b1,56)(q9,13)(u3,83)]
WRITE [(p9,43)(r1,74)]
WRITE [(i5,14)(j9,25)(u0,100)]
SHOW
WAIT 3000
READ [q8]
WRITE [(p4,22)(b2,41)(f6,92)(d9,76)]
WRITE [(h7,49)]
BACKUP
READ [o2]
READ [u3]
WRITE [(m7,4)(i6,44)(d6,82)(n1,100)]
DELETE [i5,p4,r5]
SHOW
WRITE [(k3,32)(i2,89)]
READ [q8]
WRITE [(o0,65)(h5,2)(k1,95)]
WRITE [(r0,88)(e9,39)(v8,81)(r3,36)(e9,32)]
READ [a4]
WRITE [(m8,8)(m9,73)(i0,71)(s5,94)(i3,65)]
WRITE [(t9,21)(y0,39)(g1,74)]
WAIT 1000
READ [t9]
WRITE [(h4,12)(q5,41)(w3,13)(g6,26)]
DELETE [d7,q5,k5,d6]
BACKUP
WRITE [(f8,65)]
READ [a4]
