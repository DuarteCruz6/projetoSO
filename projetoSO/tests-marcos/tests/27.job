WRITE [(n4,62)(o1,93)(t6,97)(j0,79)]
SHOW
BACKUP
WRITE [(h7,96)(r2,74)(x0,14)]
WRITE [(r6,45)]
WRITE [(w2,40)]
WRITE [(l3,31)(e3,12)]
DELETE [h7,x0,l3,j0]
WRITE [(w6,34)]
WRITE [(g2,33)(g6,95)]
WRITE [(e7,18)(h7,80)]
WRITE [(o9,87)(m4,4)(w4,35)]
READ [w4]
DELETE [t6]
WRITE [(n0,98)(a0,10)(n5,46)(l9,99)]
WRITE [(q3,78)]
WAIT 1000
WRITE [(s8,62)]
WRITE [(o2,1)(z1,76)(k9,86)]
WAIT 3000
WRITE [(d4,31)(j8,68)(v9,99)]
READ [j8]
READ [l9]
WRITE [(i4,57)(l4,5)(z8,36)]
BACKUP
BACKUP
DELETE [q3,m4]
WRITE [(u7,38)(b9,87)(e8,70)(o6,55)]
SHOW
READ [o9]
WRITE [(j3,31)(k3,8)(r9,64)(a3,72)(o7,16)]
WRITE [(i5,73)]
SHOW
READ [n5]
DELETE [k3,o6,n5]
WRITE [(f9,22)(v0,78)(x6,61)]
WRITE [(z3,45)]
WAIT 2000
DELETE [a0,b9,v9,s8]
WRITE [(j6,18)(x3,33)]
WRITE [(l7,0)(s5,21)(p3,38)(b2,41)(x4,43)]
DELETE [z3,g2]
WRITE [(h0,91)(t2,46)(z6,67)(q2,59)(n5,27)]
READ [g6]
WRITE [(p2,6)(p5,21)(l3,52)(f2,79)]
READ [x3]
WRITE [(q3,100)(w1,68)(y7,60)(n2,98)(m9,78)]
READ [b2]
