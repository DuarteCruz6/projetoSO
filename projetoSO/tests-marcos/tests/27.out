(j0, 79)
(n4, 62)
(o1, 93)
(t6, 97)
[(w4,35)]
[(j8,68)]
[(l9,99)]
(a0, 10)
(b9, 87)
(d4, 31)
(e8, 70)
(e7, 18)
(e3, 12)
(g6, 95)
(g2, 33)
(h7, 80)
(i4, 57)
(j8, 68)
(k9, 86)
(l4, 5)
(l9, 99)
(n5, 46)
(n0, 98)
(n4, 62)
(o6, 55)
(o2, 1)
(o9, 87)
(o1, 93)
(r6, 45)
(r2, 74)
(s8, 62)
(u7, 38)
(v9, 99)
(w4, 35)
(w6, 34)
(w2, 40)
(z8, 36)
(z1, 76)
[(o9,87)]
(a3, 72)
(a0, 10)
(b9, 87)
(d4, 31)
(e8, 70)
(e7, 18)
(e3, 12)
(g6, 95)
(g2, 33)
(h7, 80)
(i5, 73)
(i4, 57)
(j3, 31)
(j8, 68)
(k3, 8)
(k9, 86)
(l4, 5)
(l9, 99)
(n5, 46)
(n0, 98)
(n4, 62)
(o7, 16)
(o6, 55)
(o2, 1)
(o9, 87)
(o1, 93)
(r9, 64)
(r6, 45)
(r2, 74)
(s8, 62)
(u7, 38)
(v9, 99)
(w4, 35)
(w6, 34)
(w2, 40)
(z8, 36)
(z1, 76)
[(n5,46)]
[(g6,95)]
[(x3,33)]
[(b2,41)]
